packages/shared/src/components/tutorial/TutorialGuide.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import React, { ReactElement, ReactNode } from 'react';
+import { IconSize } from '../Icon';
+import DrawnArrowIcon from '../icons/DrawnArrow';
+
+type TutorialGuideProps = {
+  className?: string;
+  arrowPlacement?: 'top' | 'right' | 'bottom' | 'left';
+  children: ReactNode;
+};
+
+const positionMapClassName = {
+  top: 'flex-col',
+  right: 'flex-row-reverse',
+  bottom: 'flex-col-reverse',
+};
+const arrowMapClassName = {
+  right: '-mt-6 -scale-x-100',
+  bottom: 'mt-1 rotate-180',
+  left: '-mt-6',
+};
+
+const TutorialGuide = ({
+  className,
+  arrowPlacement = 'left',
+  children,
+}: TutorialGuideProps): ReactElement => {
+  return (
+    <div
+      className={classNames(
+        'flex justify-center items-center',
+        positionMapClassName[arrowPlacement],
+        className,
+      )}
+    >
+      <DrawnArrowIcon
+        className={arrowMapClassName[arrowPlacement]}
+        size={IconSize.Large}
+      />
+      {typeof children === 'string' ? (
+        <span className="typo-title2 text-theme-label-primary">{children}</span>
```

Comment:
Mobile it renders `title3` I see in the design now

Reply:
Oh, this must have been changed sometime yesterday, I'll update.